runtime/polkadot/constants/Cargo.toml

```
@@ -17,6 +17,7 @@ sp-core = { git = "https://github.com/paritytech/substrate", default-features =
 [features]
 default = ["std"]
 std = [
+	"runtime-common/std",
```

While you're at it :smile: 

```suggestion
	"runtime-common/std",
	"frame-support/std",
	"primitives/std",
```

---

Done :) I wasn't sure if this is needed. Just  did the bare minimum  for the tests to run.